### PR TITLE
[Doc] Add topic-level APIs for deduplication snapshot interval, inactive topic policies and offload policies

### PR DESCRIPTION
**Pull request by momo-jun**
### Modifications
Add the following contents to align doc with code PRs. 
1. For https://github.com/apache/pulsar/pull/8474: add a parameter `brokerDeduplicationSnapshotIntervalSeconds` to the broker configurations and cookbook.
2. For https://github.com/apache/pulsar/pull/8552: add a section of APIs to configure topic-level de-duplication snapshot interval.
3. For https://github.com/apache/pulsar/pull/7986: add a section of APIs to configure topic-level inactive topic policies.
4. For https://github.com/apache/pulsar/pull/7883: add a section of APIs to configure topic-level offload.

### Preview
image

image

image

### Documentation
  
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @315157973 @codelipenghui @Anonymitaet, can you please review this doc PR? Thanks.

Please kindly note that the same changes will be applied to more historical versions since 2.7.0 as soon as it gets approval.

File: site2/docs/admin-api-topics.md [hunk 1]
```diff
@@ -973,6 +973,90 @@ admin.topics().getBacklogSizeByMessageId(topic, messageId);
...
+
+#### Get deduplication snapshot interval
+
+To get the topic-level deduplication snapshot interval, use one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Pulsar-admin API-->
+
+```
+$ pulsar-admin topics get-deduplication-snapshot-interval options
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:987]:
```suggestion
pulsar-admin topics get-deduplication-snapshot-interval options
```

It's easier for users to copy the command without $. Please check all occurrences in this PR.

**Review by Anonymitaet — DISMISSED** [review]
Approved from the technical writing pespective

File: site2/docs/admin-api-topics.md [hunk 2]
```diff
@@ -973,6 +973,240 @@ admin.topics().getBacklogSizeByMessageId(topic, messageId);
...
+
+#### Remove offload policies
+
+To remove the topic-level offload policies, use one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Pulsar-admin API-->
+
+```
+pulsar-admin topics remove-offload-policies options
```
Comment (315157973) [inline site2/docs/admin-api-topics.md:1193]: It looks like "remove" doesn't need any options, just pass in the topic name

**Review by 315157973 — APPROVED** [review]

File: site2/docs/admin-api-topics.md (hunk 2 above)
Comment (momo-jun) [reply site2/docs/admin-api-topics.md:1193]: Thanks for your comment. However, the Pulsar admin API doc shows the "options" in the remove command. Which one shall we keep using consistently?
https://pulsar.apache.org/tools/pulsar-admin/2.10.0-SNAPSHOT/#-em-remove-offload-policies-em--89

**Pull request merged**